test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -677,19 +684,34 @@ fi
 
 if [ "$patch_version" = true ]; then
     for b in ${patch_version_upgrade_downgrade_release_branches[@]}; do
-        # Build the tip of the release branch and run test to genearte data files
+        # Build the tip of the release branch and run test to generate data files
         (build_branch $b)
         (run_test_checkpoint "$b" "row")
 
         # Pick a patch version from the list of patch versions for the release branch
         cd $b; pv=$(pick_a_version $b); cd ..
+
         (build_branch $pv)
+        rtn=$(is_test_checkpoint_recovery_supported $pv)
+        patch_fix_included=$(git log --oneline --grep=WT-8708 -b "$pv" --)
+
+        # Apply patch fix from WT-8708 to already released compatible versions to avoid test/checkpoint setting commit timestamp less than stable timestamp
+        if [ $rtn == "yes" ] && [ ! $patch_fix_included ]; then
+            cd $pv;
+
+            git format-patch -1 d4b0ad6cacb874fdc20bcc76311d789dd5a01441;
+            patch -p1 < 0001-WT-8708-Fix-timestamp-usage-error-in-test-checkpoint.patch;
+
+            (build_branch $pv)
```

Comment:
Should this line be moved down below line 706, as `build_branch` is expected to be called outside of `$pv` folder?

Reply:
Have done that!